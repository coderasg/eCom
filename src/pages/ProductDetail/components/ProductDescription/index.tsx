import React from 'react';

import styles from './productDescription.module.scss';

type Props = {
  decription: string;
};

function ProductDescription(props: Props) {
  const { decription } = props;

  return (
    <div data-testid="ProductDescription-wrapper" className={styles.productDescription}>
      <div className={styles.title}>{'Product Description'}</div>
      <div className={styles.decription}>{decription}</div>
    </div>
  );
}

export default ProductDescription;
