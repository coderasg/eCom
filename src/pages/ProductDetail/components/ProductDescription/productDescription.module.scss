@use "./../../../../variables";

.productDescription {
    border: 1px solid #ccc;


  .title {
    background-color: variables.$backgroundColor;
    padding: 10px;
  }

  .decription {
    padding: 10px;
  }
}
