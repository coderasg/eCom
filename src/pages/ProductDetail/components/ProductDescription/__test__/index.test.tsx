import React from 'react';
import renderer from 'react-test-renderer';

import ProductDescription from '../index';

describe('(Component) ProductDescription', () => {
  const defaultProps = {
    decription: 'Sample',
  };
  test('should pass snapshot test', () => {
    const wrapper = renderer.create(<ProductDescription {...defaultProps} />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
