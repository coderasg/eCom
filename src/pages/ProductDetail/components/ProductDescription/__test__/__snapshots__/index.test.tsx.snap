// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) ProductDescription should pass snapshot test 1`] = `
<div
  data-testid="ProductDescription-wrapper"
>
  <div>
    Product Description
  </div>
  <div>
    Sample
  </div>
</div>
`;
